tests/scripts/check_files.py

```diff
@@ -285,6 +285,7 @@ class UnicodeIssueTracker(LineIssueTracker):
         '\u2070\u2071\u2074-\u208E\u2090-\u209C', # Superscripts and Subscripts
         '\u2190-\u21FF', # Arrows
         '\u2200-\u22FF', # Mathematical Symbols
+        '\u2500 \u2514 \u251C' # Box Drawings characters used in markdown trees
```

Comment:
You might as well allow the whole range.

But are these characters supported in the fonts used by various systems we use with markdown files? At least direct viewing on GitHub, readthedocs, pandoc generating HTML and pandoc generating PDF.

Reply:
[x] **GitHub Flavored Markdown:** Markdown tree rendered correctly in the description
[x] **readthedocs:** Checked by building `mbedtls-docs` locally with Box Drawings characters
[x] **pandoc MD --> HTML:** Checked locally

`pandoc` has two mechanisms to convert markdown to pdf; via LaTeX or HTML. Converting via HTML will definitely work. Converting via LaTeX will work using `xelatex` and a target font that supports Box Drawings characters. Note that the default Latin Modern font does not. If we're willing to convert to PDF via HTML then we can allow these characters.

Reply:
I've allowed the whole Box Drawings range